common/asyncworkflow/queue/consumer/default_consumer.go

```diff
@@ -227,7 +227,7 @@ func (c *DefaultConsumer) processRequest(logger log.Logger, request *sqlblobs.As
 		yarpcCallOpts := getYARPCOptions(request.GetHeader())
 		scope := c.scope.Tagged(metrics.DomainTag(startWFReq.GetDomain()))
 		var resp *types.StartWorkflowExecutionResponse
-		op := func() error {
+		op := func(ctx1 context.Context) error {
```

Comment:
all the `op` implementations must use the passed in context. let's make sure we don't miss that.